product/get-product-by-channel.go

```diff
@@ -176,7 +176,7 @@ func GetLatestActiveProduct(productName, channel string, networkNode *models.Net
 	return product
 }
 
-func GetAgeInfo(productName, productVersion, channel string) (int, int) {
+func GetLifeAgeInfo(productName, productVersion, channel string) (int, int) {
```

Comment:
Is it necessary to have this function as public?

Reply:
Yes, it is used by the reserved domain (reserved/life.go)